examples/Boost-custom-args/config.cmake

```diff
@@ -2,6 +2,6 @@ hunter_config(
     Boost
     VERSION ${HUNTER_Boost_VERSION}
     CMAKE_ARGS
-    CONFIG_MACRO=BOOST_REGEX_MATCH_EXTRA;BOOST_MPL_CFG_NO_PREPROCESSED_HEADERS
```

Comment:
Keep the original example, it's still valid. Please add unit test for the script instead.

I guess the closest example will be this test with `CMAKE_ARGS` -> "etalon" comparing:
* https://github.com/ruslo/hunter/blob/b818355f23a8d90541a3843eb867019810235484/tests/hunter_create_args_file/CMakeLists.txt

You can run it similar to regular example with `PROJECT_DIR=tests/<name-of-the-test>`.